Package.swift

```diff
@@ -1,23 +1,34 @@
 // swift-tools-version: 5.10
 import PackageDescription
 
+var dependencies: [Package.Dependency] = []
+var plugins: [Target.PluginUsage]?
+
+#if os(macOS)
```

Comment:
I’m not sure what #if os means in this context. 

The Package.swift manifest will always be resolved in a Mac.

Reply:
The current CI workflow runs both on macOS and Linux. The `#if os` check is present so we don't get a build error on Linux due to SwiftLint only having a binary available for macOS.